inc/Classes/Auth.php

```diff
@@ -332,7 +332,7 @@ public function login($email, $password, $show_confirmation = 1)
                 $func->log_event(t('Login fehlgeschlagen. Email (%1) nicht verifiziert', $user['email']), "2", "Authentifikation");
 
             // User login and wrong password?
-            } elseif ($user["user_login"] and $tmp_login_pass != $user["password"]) {
+            } elseif ($user["user_login"] and !PasswordHash::verify($password, $user["password"])) {
```

Comment:
Is the variable `$tmp_login_pass` still relevant?
In line 236, we still do md5 hashing, but continue to use the clear text password.